### ISSUE TITLE
Serios bug - running second instance of "ng serve" does not work correctly.

### ISSUE THREAD

**Issue by Alex-DE-74** [angular/angular-cli#9597]
**Problem:**
Spent a lot of time just to realize, why in newly-created project a simply ng-click="someting()" and
... $rootScope.something = function(){alert()} as controller method....
do.... NOTHING, no empty message box! And that is very crazy.

After I got some more strange errors, like 404 ERR ABORTED about some scripts, which ARE NOT PART OF CURRENTLY RUNNING PROJECT, I've tried to shutdown an other "ng serve" instance for other project on the other port.

Now, the "magic" is happen - see empty message box.

That should be fixed, because that prevent from running two project in parallel and creates random undefined errors.

But the whole project does not work either, see #9596 , now get zone.js:2935 GET http://localhost:3456/login.html 404 (Not Found)
modal dismissed.

**Comment by hansl** [comment]
Hi @Alex-DE-74,

It looks like you're trying to build an Angular 1 application (mention of `ng-click` and `$rootScope`)?  Even though it should work, it requires a lot of customization of our configuration and we do not officially support that use case; the Angular CLI is purposely made for Angular 2+.

Please also note that errors in your browser can sometimes come from extensions you have installed. Make sure that those errors come from your application before reporting errors with us.

I'm going to close this issue as a support. This issue tracker is not suitable for support requests, please repost your issue on [StackOverflow](http://stackoverflow.com/) using tag `angular`.

If you are wondering why we don't resolve support issues via the issue tracker, please [check out this explanation](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-got-a-question-or-problem).

**Comment by Alex-DE-74** [comment]
Hi @hansl 

Thank you for clarification. Does this implies, all code with AngularJS is dead and must be rewritten if angular-cli is used? Any alternatives for Windows?

An yes, the errors cames **not** from extension. The second ng instance runs correctly (this was very small test) only if first one (served other project) was closed. I also tried onclick - no click event was noified at all as long, as other ng was running.
